src/store/index.js

```diff
@@ -0,0 +1,22 @@
+import { createStore, applyMiddleware, compose } from 'redux'
+import createSagaMiddleware, { END } from 'redux-saga'
+import rootReducer from  "../reducers"
+
+export default function configureStore(initialState) {
+  const sagaMiddleware = createSagaMiddleware()
+
+  const store = createStore(
+    rootReducer,
+    initialState,
+    compose(
+      applyMiddleware(
+        sagaMiddleware
+      ),
+      window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
+    )
+  )
+
+  store.runSaga = sagaMiddleware.run
```

Comment:
It is a bad practice to 'enhance' the store this way. Calling `runSaga` on the saga middleware is a better approach.